docs/concepts/manifest-builds.md

```diff
@@ -134,6 +137,32 @@ What this means in practice is:
 - Man pages should be placed in `$out/share/man`.
 - Libraries should be placed in `$out/lib`
 
+### Sandboxed builds
+
+The script that you specify in `command` is run on your host machine by default.
+This means that it may build against dependencies that are outside of your
```

Comment:
```suggestion
This means that it may build against dependencies that are outside of your
```
Not just dependencies, but also caches, previous build artifacts, etc.